Sources/Purchasing/Purchases/Purchases.swift

```diff
@@ -253,7 +253,7 @@ public typealias StartPurchaseBlock = (@escaping PurchaseCompletedBlock) -> Void
     private let receiptFetcher: ReceiptFetcher
     private let requestFetcher: StoreKitRequestFetcher
     private let paymentQueueWrapper: EitherPaymentQueueWrapper
-    private let systemInfo: SystemInfo
+    fileprivate let systemInfo: SystemInfo
```

Comment:
Why did this need to be changed to `fileprivate`?

Reply:
The deprecated method `finishTransactions` uses this, so when it was moved to a deprecated extension, this needed its access level changed. The choices are

- Leave deprecated `finishTransactions` in the main class declaration and `systemInfo` can stay `private`.
- Move deprecated `finishTransactions` to the "deprecated" extension in this file, and make `systemInfo` `fileprivate`.
- Move deprecated `finishTransactions` to the deprecations source file, and make `systemInfo` `internal`.

Discussed this issue with others in the 4.0 PR, the middle option was where we landed, but nobody felt really strongly about it.... thoughts?